executor/builder.go

```diff
@@ -625,7 +550,6 @@ func (b *executorBuilder) buildTableScan(v *plan.PhysicalTableScan, s *plan.Sele
 	}
 	supportDesc := client.SupportRequestType(kv.ReqTypeSelect, kv.ReqSubTypeDesc)
 	if !memDB && client.SupportRequestType(kv.ReqTypeSelect, 0) {
```

Comment:
During plan building, this condition decides if we can push down the predicates.
During executor building, this condition decides to use which executor.